test/starlark_tests/macos_static_framework_tests.bzl

```diff
@@ -88,6 +92,9 @@ def macos_static_framework_test_suite(name):
         build_type = "simulator",
         compilation_mode = "opt",
         target_under_test = "//test/starlark_tests/targets_under_test/macos:static_fmwk_with_swift_and_avoid_deps",
+        build_settings = {
+            build_settings_labels.use_library_evolution: "True",
+        },
```

Comment:
@brentleyjones Do you any idea why the test is failing after setting flag to `true`? Looks like other `build_settings` are ignored as well (i.e. `test/starlark_tests/apple_static_xcframework_import_tests.bzl#L252`) - the test is passing even when I toggle the value.

Reply:
I do not.